### PR TITLE
Update action admin

### PR DESCRIPTION
**Pull request by emainguelag**
admin action update + add without PDOException manage

### CODE REVIEW

File: src/Model/ActionManager.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        `label` = :label,
+        `owner_id` = :owner_id,
+        `target_id` = :target_id
+        WHERE id=:id;
+        ";
+
+        $statement = $this->pdo->prepare($query);
+        $statement->bindValue('id', $action['id'], PDO::PARAM_INT);
+        $statement->bindValue('label', $action['label'], PDO::PARAM_STR);
+        $statement->bindValue('owner_id', $action['owner_id'], PDO::PARAM_STR);
```
Comment (aldaitzwild) [inline src/Model/ActionManager.php]: This should be a INT

File: src/Model/ActionManager.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        $statement->bindValue('target_id', $action['target_id'], PDO::PARAM_STR);
```
Comment (aldaitzwild) [inline src/Model/ActionManager.php]: Same Here

File: src/Model/ActionManager.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    {
+        $query = "INSERT INTO " . self::TABLE . " SET 
+        `label` = :label,
+        `owner_id` = :owner_id,
+        `target_id` = :target_id;
+        ";
+
+        $statement = $this->pdo->prepare($query);
+        $statement->bindValue('label', $action['label'], PDO::PARAM_STR);
+        $statement->bindValue('owner_id', $action['owner_id'], PDO::PARAM_STR);
```
Comment (aldaitzwild) [inline src/Model/ActionManager.php]: Here too

File: src/Model/ActionManager.php [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        $statement->bindValue('target_id', $action['target_id'], PDO::PARAM_STR);
```
Comment (aldaitzwild) [inline src/Model/ActionManager.php]: I guess you get the point

File: src/View/Action/admin_show_action.html.twig [hunk 5]
```diff
@@ -0,0 +1,54 @@
+{% extends 'layout.html.twig' %}
+
+{% block content %}
+	<a class="logo" href="/"><img src="/assets/images/logo.svg"></a>
+	<main class="adminMain">
+		<h1 class="adminTitle">Action
+			{{ action.id }}</h1>
+		<h2>
```
Comment (aldaitzwild) [inline src/View/Action/admin_show_action.html.twig]: I wonder where this tag h2 ends ...

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Comment by emainguelag** [comment]
5 errors corrected into the new push on this branch. Thank you!

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**